packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -192,7 +192,7 @@ void updateLocalProperties({String projectPath, BuildInfo buildInfo}) {
     settings.writeContents(localProperties);
 }
 
-Future<Null> buildGradleProject(BuildInfo buildInfo, String target, String kernelPath) async {
+Future<Null> buildGradleProject(BuildInfo buildInfo, String target, bool previewDart2) async {
```

Comment:
Is there any downside to making `previewDart2` a member of `BuildInfo`?

Reply:
None that I can see. Done.